test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -125,7 +125,7 @@ public void addCommand_emptyAddressBook_addressBookContainsPerson() {
 
         assertTrue(people.contains(p));
         assertEquals(1, people.immutableListView().size());
-        assertFalse(result.getRelevantPersons().isPresent());
+        assertTrue(result.getRelevantPersons().isPresent());
```

Comment:
This will return an assertion error? Consider that this JUnit test is for an empty Addressbook.